### PR TITLE
Implemented get_watchlist use case

### PR DESCRIPTION
**Pull request by sgspencer2618**

### CODE REVIEW

File: src/interface_adapters/get_watchlist/GetWatchlistState.java
Comment (rohanbrohan) [inline src/interface_adapters/get_watchlist/GetWatchlistState.java:1]: What is this file for?

File: src/data_access/WatchlistAccessObject.java
Comment (rohanbrohan) [inline src/data_access/WatchlistAccessObject.java:1]: Might be a good idea to make WatchlistAccessObject also implement GetWatchlistDataAccessInterface.

**Review by rohanbrohan — CHANGES_REQUESTED** [review]
Looks very good. I'll Merge once the comments are addressed.

**Comment by sgspencer2618** [comment]
> Looks very good. I'll Merge once the comments are addressed. 

roger that, I'll get to it today.

**Pull request closed without merging**